sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/diagnose.py

```diff
@@ -3,15 +3,15 @@
 # ---------------------------------------------------------
 from typing import Any, Dict, List, Optional
 
-from azure.ai.ml._restclient.v2022_12_01_preview.models import (
```

Comment:
According to previous comment, can workspace continue to use 2022_12_01_preview version?